src/skrollr.js

```diff
@@ -807,7 +810,8 @@
 
 					duration = duration * (1 - targetRatio);
 
-					_instance.animateTo((targetTop + 0.5) | 0, {easing: 'outCubic', duration: duration});
+					var direction = (distanceY)>0?'down':'up';
+					_instance.animateTo((targetTop + 0.5) | 0, {easing: 'outCubic', duration: duration, done: _listeners.afterfling(direction)});
```

Comment:
Your code doesn't work, you're executing `_listeners.afterfling(direction)` immediately.